### PR TITLE
Catch OOME during copy HTML to Clipboard #1059

### PR DESCRIPTION
**Pull request by jukzi**
Copied HTML is much bigger then plain text. Fall back to copy at least the plain text.

https://github.com/eclipse-platform/eclipse.platform.swt/issues/1059

### CODE REVIEW

**Comment by jukzi** [comment]
i have no clue how this is related or how it can be fixed: build fails with
```
Only qualifier changed for (org.eclipse.swt.gtk.linux.ppc64le/3.125.0.v20240305-1200). Expected to have bigger x.y.z than what is available in baseline (3.125.0.v20240227-1638) 
Only qualifier changed for (org.eclipse.swt.gtk.linux.x86_64/3.125.0.v20240305-1200). Expected to have bigger x.y.z than what is available in baseline (3.125.0.v20240227-1638)
```

**Comment by merks** [comment]
I'm not show one coordinates the fragment's version with the host's version

image

But the error message sounded like it was complaining about a fragment's version.

File: bundles/org.eclipse.swt/META-INF/MANIFEST.MF [hunk 1]
```diff
@@ -2,7 +2,7 @@ Manifest-Version: 1.0
 Bundle-Name: %pluginName
 Bundle-Vendor: %providerName
 Bundle-SymbolicName: org.eclipse.swt; singleton:=true
-Bundle-Version: 3.125.0.qualifier
+Bundle-Version: 3.125.200.qualifier
```
Comment (iloveeclipse) [inline bundles/org.eclipse.swt/META-INF/MANIFEST.MF:5]: `0 -> 200` is wrong. Should be `0 -> 100`.
But also all fragments need to be updated.
Please do it in dedicated PR.

**Comment by laeubi** [comment]
Only qualifier changed usually means you need to touch the `forceQualifier` instead of increment the version because you changed something indirectly.

**Comment by iloveeclipse** [comment]
In SWT the fragment versions should match SWT version, so that is OK here.

**Comment by merks** [comment]
> Only qualifier changed usually means you need to touch the `forceQualifier` instead of increment the version because you changed something indirectly.

I generally have only ever done that in features; mostly for features that include 3rd party bundles which are becoming rare.  After all the forceQualifier edit is not different than any other actual edit in the source project, right?

**Comment by laeubi** [comment]
> > Only qualifier changed usually means you need to touch the `forceQualifier` instead of increment the version because you changed something indirectly.
> 
> I generally have only ever done that in features; mostly for features that include 3rd party bundles which are becoming rare. After all the forceQualifier edit is not different than any other actual edit in the source project, right? So the force is only for the first variation of the problem, i.e., compiler change but not actual other change in the bundle.

SWT is special as the actual code is not in the bundle but in the fragments, but the fragment only include specific parts, so it could happen that you change a "specialized part" and it will be picked automatic, or you change a generic part that might require manual intervention.
